File: cpp/bc3346.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
	double fPib, sPib, res;
	
	cin >> fPib >> sPib;
	
	fPib = fPib / 100 + 1;
	sPib = sPib / 100 + 1;
	
	res = (fPib * sPib - 1) * 100;
	
	cout << fixed << setprecision(6) << res << endl;
}

File: python/bc3346.py
cVal = input().split(" ")

fPib = float(cVal[0]) / 100.0 + 1.0
sPib = float(cVal[1]) / 100.0 + 1.0

res = (fPib * sPib - 1) * 100.0

print(f"{res:.6f}")

